virttest/utils_test/__init__.py

```diff
@@ -922,9 +947,15 @@ def config_control(control_path, job_args=None):
     kernel_install_path = os.path.join(autotest_path, 'tests',
                                        'kernelinstall')
     kernel_install_present = os.path.isdir(kernel_install_path)
-
     autotest_basename = os.path.basename(autotest_path)
     autotest_parentdir = os.path.dirname(autotest_path)
+    client_tests_url = ("https://codeload.github.com/autotest/"
+                        "autotest-client-tests/zip/master")
+    client_tests_url = params.get("client_test_url", client_tests_url)
```

Comment:
client_tests_url is only used if `os.path.isdir` is not existing, you should probably move it there to not affect the default flow.